plugins/continuous_dataset.cc

```diff
@@ -518,13 +518,25 @@ reconstituteConfig(const MatrixNamedRow & row)
 {
     Json::Value current;
 
+    auto pruneDoubleQuotes = [] (const string & in) {
```

Comment:
Moi je cherche une string in string out. Est-ce que je peux simplement prendre les paths element et ajouter des points entre?